Tools/TestWebKitAPI/Tests/WebKitCocoa/FindInPage.mm

```diff
@@ -913,4 +913,44 @@ static void swizzledPerformTextSearchWithQueryString(id, SEL, NSString *, UIText
     EXPECT_TRUE(findSessionSupportsReplacement());
 }
 
+static bool hasPerformedTextSearchWithQueryString = false;
+
+static void swizzledPerformTextSearchInPDFWithQueryString2(id, SEL, NSString *, UITextSearchOptions *, id<UITextSearchAggregator> aggregator)
+{
+    [aggregator finishedSearching];
+    hasPerformedTextSearchWithQueryString = true;
+}
+
+TEST(WebKit, WKPDFViewFindInteractionAfterPageRefresh)
+{
+    // Swizzle out the method that performs searching, since PDFHostViewController (a remote view
+    // (controller) cannot be created in TestWebKitAPI, and we cannot actually search the PDF.
+    std::unique_ptr<InstanceMethodSwizzler> isInBackgroundSwizzler = makeUnique<InstanceMethodSwizzler>(NSClassFromString(@"WKPDFView"), @selector(performTextSearchWithQueryString:usingOptions:resultAggregator:), reinterpret_cast<IMP>(swizzledPerformTextSearchInPDFWithQueryString2));
```

Comment:
`isInBackgroundSwizzler` -> `performTextSearchInPDFWithQueryStringSwizzler`.

I must have incorrectly copied the name elsewhere, so you can update the other location too.